Documentation for Zeta Marketing Platform destination

hsaraiya opened the pull request:
Added:
* a new file `zetamarketingplatform.md` for our integration with Adobe Experience Platform
* supporting screenshots
@adobe-documentation-team, please review.

hsaraiya committed:
Create zetamarketingplatform.md

hsaraiya committed:
Updating zetamarketingplatform.md

hsaraiya committed:
Add assets for the ZMP destination

hsaraiya committed:
Added image to Configuring Destination

hsaraiya committed:
Upload image showing a mapping example

hsaraiya committed:
Add content till mapping section

hsaraiya committed:
Clarify pre-requisites section

hsaraiya committed:
Example final customer list as viewable in ZMP

hsaraiya committed:
Single Customer View in ZMP

hsaraiya committed:
Update exported data / validate data export

hsaraiya committed:
Add info on auto-synchronization of segments

hsaraiya committed:
Remove md5 details

hsaraiya committed:
Update supported identities section

hsaraiya committed:
Delete help/destinations/assets/catalog/data-management-platform/zmp-extension/ConfigureNewDestination.png

hsaraiya committed:
ConfigureNewDestination.png

Update screen for configuring new destination

hsaraiya committed:
Update SupportedIdentities

hsaraiya committed:
Delete help/destinations/assets/catalog/data-management-platform/zmp-extension/ConfigureNewDestination.png

hsaraiya committed:
Update ConfigureNewDestination.png

Correct spelling error

hsaraiya committed:
Update zetamarketingplatform.md

minor updates

hsaraiya committed:
Update zetamarketingplatform.md

add link to destination details

hsaraiya commented:
Adobe CLA signed

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+## Overview {#overview}
+
+The Zeta Marketing Platform, henceforth ZMP, is a cloud-based system with the tools to help you acquire, grow, and retain customers more efficiently, powered by intelligence (proprietary data and AI). For more details, refer to [Zeta Global](https://zetaglobal.com/). 
```
Please avoid phrasing like "henceforth" in documentation. Let's rephrase that part as "The Zeta Marketing Platform (ZMP) is a cloud-based system which helps you acquire [...]"

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+---
+title: Zeta Marketing Platform
+description: The Zeta Marketing Platform, henceforth ZMP, is a cloud-based system with the tools to help you acquire, grow, and retain customers more efficiently, powered by intelligence (proprietary data and AI). For more details, refer to [Zeta Global](https://zetaglobal.com/). 
```
Please avoid phrasing like "henceforth" in documentation. Let's rephrase that part as "The Zeta Marketing Platform (ZMP) is a cloud-based system which helps you acquire [...]"

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+## Use cases {#use-cases}
+
+### Use case #1 {#use-case-1}
```
Please update the use case headings to something that gives readers an overview of what the use case is about.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+The Zeta Marketing Platform, henceforth ZMP, is a cloud-based system with the tools to help you acquire, grow, and retain customers more efficiently, powered by intelligence (proprietary data and AI). For more details, refer to [Zeta Global](https://zetaglobal.com/). 
+
+With the ZMP connector available in Adobe Experience Platform, you can seamlessly synchronize your audiences from Experience Platform to the ZMP. 
```
Is the destination card in the Experience Platform UI called "ZMP"? If not, please refer to the connector by its full name (Zeta Marketing Platform).

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+A marketer wants to build unique audience profiles, identify their most valuable segments, and use them across any digital channels that the ZMP supports. They want to create a true 360 view of a consumer profile, build and activate meaningful audiences. More details on which channels the ZMP supports can be found [here](https://zetaglobal.com/platform/integrations/).
+
+### Use case #2 {#use-case-2}
+An advertiser wants to target users in audiences with ads through the Zeta DSP as the user interacts with their brands. More information on the Zeta DSP and other DSPs that the ZMP supports can be found [here](https://knowledgebase.zetaglobal.com/programmatic-user-guide/).
```
I strongly suggest replacing ZMP with Zeta Marketing Platform. This phrase sounds like a convoluted collection of acronyms which will confuse readers (More information on the Zeta DSP and other DSPs that the ZMP supports...).

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+### Zeta Marketing Platform prerequisites
+
+* Prior to setting up a new destination, please ensure the presence of a pre-existing shell customer list in your ZMP account. You will  be required to choose one of these customer lists as the designated target to receive the Adobe Experience Platform audience that you plan to send. You can create an empty customer list in the ZMP by following the instructions [here](https://knowledgebase.zetagl ...
```
The first sentence in the prerequisites can be improved. What does "pre-existing shell customer list" mean? Does it mean "you need to create an empty customer list in your Zeta Marketing Platform account"?

If so, please rephrase that part along the lines of: "Before you set up a new connection to the Zeta Marketing Platform destination, you must create an empty customer list in your Zeta Marketing Platform account."

vfalon replied:
Extra space in `"You will  be required"`

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+* Prior to setting up a new destination, please ensure the presence of a pre-existing shell customer list in your ZMP account. You will  be required to choose one of these customer lists as the designated target to receive the Adobe Experience Platform audience that you plan to send. You can create an empty customer list in the ZMP by following the instructions [here](https://knowledgebase.zetagl ...
+* Although the Adobe Experience Platform permits the activation of multiple audiences to a particular ZMP destination instance, it is mandatory that each ZMP destination instance receives only one Experience Platform audience. To handle multiple audiences from the Experience Platform, create additional ZMP instances for each audience and select a different shell customer list from the dropdown. T ...
+* The username and password needed to configure the destination will be **api** and the **REST API Key**, which you can view in your ZMP account by navigating to **Settings** > **Integrations** > **Keys & Apps** section. More information can be found [here](https://knowledgebase.zetaglobal.com/zmp/integrations).
```
Always use present tense in documentation, and use simple, short, phrases whenever possible:

Use the following credentials to configure the destination:

- Username: `api`
- Password:  Your ZMP REST API Key.  You can find your REST API Key by logging in to your ZMP account and navigating to **Settings** > **Integrations** > **Keys & Apps** section. See the [ZMP documentation]((https://knowledgebase.zetaglobal.com/zmp/integrations) for more details.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+## Supported identities {#supported-identities}
+
+The ZMP destination connector will allow you to map platform identities to identities that the ZMP accepts. 
```
This sentence can be removed. It doesn't add any information to what is already below it.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+The ZMP destination connector will allow you to map platform identities to identities that the ZMP accepts. 
+>[!IMPORTANT]
+> Please note that it is mandatory to map a source identity namespace to ZMP's `uid` target identity so that the ZMP can uniquely differentiate each profile. 
```
Move this note just above the table and after the introductory sentence.

vfalon replied:
Avoid "please" in product documentation. Let's rephrase this note:

The ZMP destination requires you to map a source identity namespace to the ZMP `uid` target identity. This helps the Zeta Marketing Platform uniquely differentiate each profile.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+|Target Identity|Description|Considerations|Notes|
+---------|----------|----------|----------|
+|uid|Unique id that ZMP uses to differentiate customer profiles|Mandatory|Choose the `Email` standard identity namespace if you want to identify unique profiles using their email addresses. Alternatively, you can opt to map your custom namespace to `uid` if customer profiles do not have an email.|
```
When referring to an ID as a noun, it must be capitalized:  "Unique ID that ZMP uses to..."

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+| Audience origin | Supported | Description | 
+---------|----------|----------|
+| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md).|
```
Please leave the custom audiences entry in this table, as shown in the template: https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#supported-identities.

If it's not supported, mark it with an X in the Supported column, but don't remove the entry from the table.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+* Enter the **[!UICONTROL Username]** as **api**
```
- **[!UICONTROL Username]**: `api`
- **[!UICONTROL Password]**: Your ZMP REST API Key. You can find your REST API Key by logging in to your ZMP account and navigating to Settings > Integrations > Keys & Apps section. See the [ZMP documentation]((https://knowledgebase.zetaglobal.com/zmp/integrations) for more details.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+![Image showing ZMP configuration](../../assets/catalog/data-management-platform/zmp-extension/ConfigureNewDestination.png)
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL ZMP Account Site Id]**: Your ZMP **Site Id** where you want to send your audiences to. You can view your Site Id by navigating to **Settings** > **Integrations** > **Keys & Apps** section. More information can be found [here](https://knowledgebase.zetaglobal.com/zmp/integrations).
```
Please update **[!UICONTROL ZMP Account Site Id]** to **[!UICONTROL ZMP Account Site ID]**.

Make sure to update the name in your destination configuration submission, too (and take another screenshot showing the correct casing for ID).

hsaraiya replied:
@vfalon The identity is shown in the ZMP UI as `Id` (lowercase D). I'd prefer keeping it that way. Thoughts?

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+![Image showing ZMP configuration](../../assets/catalog/data-management-platform/zmp-extension/ConfigureNewDestination.png)
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL ZMP Account Site Id]**: Your ZMP **Site Id** where you want to send your audiences to. You can view your Site Id by navigating to **Settings** > **Integrations** > **Keys & Apps** section. More information can be found [here](https://knowledgebase.zetaglobal.com/zmp/integrations).
```
Your ZMP **Site Id** where you want to send your audiences to.  ->>> Your ZMP **Site ID** where you want to send your audiences to.

hsaraiya replied:
@vfalon We refer to site identity as `Site Id` (lowercase D) in our platform. Is this change needed?

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+![Image showing ZMP configuration](../../assets/catalog/data-management-platform/zmp-extension/ConfigureNewDestination.png)
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL ZMP Account Site Id]**: Your ZMP **Site Id** where you want to send your audiences to. You can view your Site Id by navigating to **Settings** > **Integrations** > **Keys & Apps** section. More information can be found [here](https://knowledgebase.zetaglobal.com/zmp/integrations).
```
You can view your Site Id by navigating -->>>  You can view your Site ID by navigating

hsaraiya replied:
@vfalon We refer to site identity as `Site Id` (lowercase D) in our platform. Is this change needed?

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL ZMP Account Site Id]**: Your ZMP **Site Id** where you want to send your audiences to. You can view your Site Id by navigating to **Settings** > **Integrations** > **Keys & Apps** section. More information can be found [here](https://knowledgebase.zetaglobal.com/zmp/integrations).
+*  **[!UICONTROL ZMP Segment]**: The customer list segment in your ZMP Site Id account that you want to be updated with the Platform audience.
```
list segment in your ZMP Site Id account --->>> list segment in your ZMP Site ID account

hsaraiya replied:
@vfalon We refer to site identity as `Site Id` (lowercase D) in our platform. Is this change needed?

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+Below is an example of correct identity mapping when exporting profiles to [!DNL Zeta Marketing Platform].
+
+Selecting source fields:
+* Select an identity namespace (custom or standard such as `Email`) as source field that uniquely identifies a profile in Adobe Experience Platform and [!DNL Zeta Marketing Platform].
```
Select a source identity namespace (custom or standard, such as `Email`) that uniquely identifies a profile in Adobe Experience Platform and [!DNL Zeta Marketing Platform].

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+* Select any XDM source profile attributes that need to be exported to and updated in the [!DNL Zeta Marketing Platform].
+
+Selecting target fields:
+* (Mandatory) Select `uid` as the target identity that is mapped to the source identity namespace. 
```
Phrasing issue here. Please update the first point to:

* (Mandatory) Select `uid` as the target identity to which you map a source identity namespace.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+Selecting target fields:
+* (Mandatory) Select `uid` as the target identity that is mapped to the source identity namespace. 
+* (Optional) Select `email_md5_id` as the target identity that is mapped to the source identity namespace that represents email md5 values. In this scenario, map `uid` to a source identity namespace.
```
(Optional) Select `email_md5_id` as the target identity that is mapped to the source identity namespace that represents email md5 values. In this scenario, map `uid` to a source identity namespace.

Update to: 
(Optional) Select `email_md5_id` as the target identity to which you mapped the source identity namespace that represents email md5 values.


I don't understand the point of this sentence "In this scenario, map `uid` to a source identity namespace." My understanding is that `uid` mapping is mandatory either way. You already mentioned that at the previous bullet point.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+Selecting target fields:
+* (Mandatory) Select `uid` as the target identity that is mapped to the source identity namespace. 
+* (Optional) Select `email_md5_id` as the target identity that is mapped to the source identity namespace that represents email md5 values. In this scenario, map `uid` to a source identity namespace.
+* Select any other destination profile attribute names that need to be mapped to corresponding XDM source profile attributes.
```
Please rephrase this to "Select any additional target mappings if needed."

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+## Exported data / Validate data export {#exported-data}
+
+A successful audience syndication from experience platfrom to the ZMP will update the target customer list in the ZMP. The count as well as sample profiles in the target customer list will be exactly equal to the number of identities that were successfuly activated from experience platform as shown below.
```
A successful audience syndication from experience platfrom to the ZMP will update the target customer list in the ZMP. The count as well as sample profiles in the target customer list will be exactly equal to the number of identities that were successfuly activated from experience platform as shown below.

Please rephrase to:

A successful audience activation from Experience Platform to the ZMP updates the target customer list in the ZMP. The count and the sample profiles in the target customer list will be equal to the number of identities that were successfully activated.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+
+![Customer List in ZMP](../../assets/catalog/data-management-platform/zmp-extension/CustomerListInZMP.png)
+
+Each audience member that was syndicated from the experience platform will also be visible under **Audiences** > **People** in the ZMP. 
```
Each audience member that was syndicated from the experience platform will also be visible under **Audiences** > **People** in the ZMP. 

Rephrase as:

Each audience member that was activated from Experience Platform will also be visible under **Audiences** > **People** in the ZMP.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, read the [Data Governance overview](/help/data-governance/home.md).
+
+## Additional resources {#additional-resources}
+* [Zeta Knowledge Base](https://knowledgebase.zetaglobal.com/zmp/).
```
Remove the full stop at the end of this item.

vfalon commented on help/destinations/catalog/data-management/zetamarketingplatform.md:
```diff
@@ -0,0 +1,144 @@
+---
+title: Zeta Marketing Platform
```
A couple of file management issues:

- Please rename the doc page to `zeta-marketing-platform.md`.
- Please rename the `assets/catalog/data-management-platform/zmp-extension/` folder to `assets/catalog/data-management-platform/zeta-marketing-platform/`

vfalon reviewed (changes requested):
@hsaraiya thank you for the docs submission! There are some changes that we need you to address before we move forward with this destination. Please see my comments and let me know if you have any questions.

Looking forward to see the updates!

hsaraiya committed:
Address doc review feedback

- Update and rename zetamarketingplatform.md to zeta-marketing-platform.md
- make text changes

hsaraiya committed:
Rename zmp-externsion to zeta-marketing-platform

hsaraiya committed:
Update assets

hsaraiya commented:
> @hsaraiya thank you for the docs submission! There are some changes that we need you to address before we move forward with this destination. Please see my comments and let me know if you have any questions.
> 
> Looking forward to see the updates!

@vfalon Thank you for the exhaustive list and detailed recommendations. I have made all the changes requested except the one with Id v/s ID. I haven't "resolved" comments just in case you'd want to review.

vfalon commented on help/destinations/catalog/data-management/zeta-marketing-platform.md:
```diff
@@ -0,0 +1,144 @@
+
+### Zeta Marketing Platform prerequisites
+
+* Before you set up a new connection to the Zeta Marketing Platform destination, you must create an empty customer list in your Zeta Marketing Platform account. You will be required to choose one of these customer lists as the designated target to receive the Adobe Experience Platform audience that you plan to send. You can create an empty customer list in the ZMP by following the instructions [h ...
```
"You will be required to choose one of these customer lists .... "

Please rephrase as "You must choose one of these customer lists...

vfalon commented on help/destinations/catalog/data-management/zeta-marketing-platform.md:
```diff
@@ -0,0 +1,144 @@
+|Target Identity|Description|Considerations|Notes|
+---------|----------|----------|----------|
+|uid|Unique ID that ZMP uses to differentiate customer profiles|Mandatory|Choose the `Email` standard identity namespace if you want to identify unique profiles using their email addresses. Alternatively, you can opt to map your custom namespace to `uid` if customer profiles do not have an email.|
+|email_md5_id|Email MD5 that represents each customer profile|Optional|Choose this target identity when you aim to uniquely identify customer profiles using email MD5 values. In this case, set `uid` (mandatory) to either the same email MD5 values or another appropriate identity namespace.|
```
Regarding the MD5 mapping: please add a sentence here explaining that if customers want to map email addresses to MD5, they must have the email addresses already in MD5 format in Experience Platform, since Platform does not convert plain text to MD5.

vfalon commented on help/destinations/catalog/data-management/zeta-marketing-platform.md:
```diff
@@ -0,0 +1,144 @@
+{style="table-layout:auto"}
+
+>[!NOTE]
+> As individual members are added or removed from the platform audience, updates will be sent to the ZMP to ensure that the destination customer list is synchronized accordingly.
```
Please capitalize Platform wherever it is referenced. Or use Experience Platform for even more clarity.

vfalon commented on help/destinations/catalog/data-management/zeta-marketing-platform.md:
```diff
@@ -0,0 +1,144 @@
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+* **[!UICONTROL Username]**: `api`
+* **[!UICONTROL Password]**: Your ZMP REST API Key. You can find your REST API Key by logging in to your ZMP account and navigating to **Settings** > **Integrations** > **Keys & Apps** section. See the [ZMP documentation]((https://knowledgebase.zetaglobal.com/zmp/integrations) for more details.
```
` See the [ZMP documentation](https://knowledgebase.zetaglobal.com/zmp/integrations) for more details.`

There's an extra ( there that breaks the link.

vfalon commented on help/destinations/catalog/data-management/zeta-marketing-platform.md:
```diff
@@ -0,0 +1,144 @@
+
+Selecting target fields:
+* (Mandatory) Select `uid` as the target identity to which you map a source identity namespace. 
+* (Optional) Select `email_md5_id` as the target identity to which you mapped the source identity namespace that represents email md5 values.
```
On the row regarding MD5 mapping, please add a sentence explaining that if customers want to map email addresses to MD5, they must have the email addresses already in MD5 format in Experience Platform, since Platform does not convert plain text to MD5.

vfalon commented on help/destinations/catalog/data-management/zeta-marketing-platform.md:
```diff
@@ -0,0 +1,144 @@
+* (Optional) Select `email_md5_id` as the target identity to which you mapped the source identity namespace that represents email md5 values.
+* Select any additional target mappings if needed.
+
+![Identity mapping](../../assets/catalog/data-management-platform/zeta-marketing-platform/MappingExample.png)
```
Please follow the image naming format described in the authoring guide(https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/authoring-best-practices.html?lang=en#linking). 

Use: yourdestination-authentication-details.png, yourdestination-destination-details.png

Instead of `MappingExample.png` please use `mapping-example.png`.

Review your assets folder and change all the image names to the correct format.

vfalon reviewed (changes requested):
@hsaraiya thanks for making the updates! There are a few more changes to make, but otherwise it looks great! Looking forward to see the final updates. Thanks!

hsaraiya committed:
Update text in zeta-marketing-platform.md

hsaraiya committed:
Rename ConfigureNewDestination.png to zeta-configure-new-destination.png

hsaraiya committed:
Rename SingleCustomerViewInZMP.png to zeta-single-customer-view-in-zmp.png

hsaraiya committed:
Rename zeta-single-customer-view-in-zmp.png.png to zeta-single-customer-view-in-zmp.png

hsaraiya committed:
Rename MappingExample.png to zeta-mapping-example.png

hsaraiya committed:
Rename CustomerListInZMP.png to zeta-customer-list-in-zmp.png

hsaraiya committed:
Update assets' URLs

hsaraiya commented:
@vfalon I have made all the above requested changes. Thank you!
